### DISCUSSION TITLE
Set custom styling for Accordion Panel when opened

### DISCUSSION THREAD

**Discussion by Andyy58** [mantinedev/mantine#6340] [Q&A] [answered]
Hi, I'm looking for a way to set custom styling for the accordion component when it is expanded vs when it is closed. Can't seem to find anything in the documentation about this specifically. Are there any straightforward ways to do this?

Specifically I'd like to change the bg color of the accordion item and underline the label when it is expanded.

Thanks!

**Comment by Sajarin-M** [comment, marked as answer]
You can use the `data-active` attribute in Accordion item for this. 
Refer the second example in [this section](https://mantine.dev/core/accordion/#styles-api) of Mantine docs
